projects/angular-d3-cloud/src/lib/angular-d3-cloud.component.ts

```diff
@@ -135,6 +136,10 @@ export class AngularD3CloudComponent implements OnChanges, OnInit {
       throw new TypeError(`${AngularD3CloudComponent.TAG}: [fontSizeMapper] must be a function. Current value is: [${this.fontSizeMapper}]`)
     }
 
+    if (!this.fillMapper || typeof this.fillMapper !== 'function') {
```

Comment:
follow up with above comment then this will not be required, only check to see if its type is `function` is ok

Reply:
ok